### DISCUSSION TITLE
`create-svelte` with javascript & jsdoc setup installs typescript dependencies

### DISCUSSION THREAD

**Discussion by bnn1** [sveltejs/kit#11612] [Q&A] [answered]
Hi guys. 

When I created a new svelte kit project with `pnpm create svelte@latest` with the `Yes, using JavaScript with JSDoc comments` option, it added typescript dependencies (`typescript`, `@types/eslint`) to the project. 

Are typescript dependencies required, or is it a bug?

**Comment by danielniccoli** [comment, marked as answer]
The question you answered with

> Yes, using JavaScript with JSDoc comments

was 

> Add type checking with __TypeScript__?
